js/select2.js

```diff
@@ -46,12 +46,10 @@
         if (Object.prototype.hasOwnProperty.call(config, 'ajax')) {
           var $select = $(this);
           var $list = $(this).next('.select2-container').find('ul.select2-selection__rendered');
-          $list.sortable({
-            placeholder: 'ui-state-highlight select2-selection__choice',
-            forcePlaceholderSize: true,
-            items: 'li:not(.select2-search__field)',
-            tolerance: 'pointer',
-            stop: function () {
+          Sortable.create($list[0], {
+            draggable: 'li:not(.select2-search)',
+            forceFallback: true,
```

Comment:
I created a small demo at https://jsbin.com/rexadatola/edit?html,js,output
This seems to be a general problem with the combination of select2 and sortablejs.
We might want to add issues to these projects.